framework/src/media/audio/audio_manager.h

```diff
@@ -376,6 +436,149 @@ audio_manager_result_t set_input_audio_gain(uint8_t gain);
  ****************************************************************************/
 audio_manager_result_t set_output_audio_volume(uint8_t volume);
 
+/****************************************************************************
+ * Name: foreach_supported_process_device
+ *
+ * Description:
+ *   Find stream in device that is supports specific process on dsp
+ *
+ * Input parameter:
+ *   card_id : actual card id to be returned, device_id : founded device id,
+ *   subtype : supported sub type of processing, type : supported type of processing
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t find_stream_in_device_with_process_type(device_process_type_t type, int *card_id, int *device_id, device_process_subtype_t *subtype);
+
+/****************************************************************************
+ * Name: set_stream_in_policy
+ *
+ * Description:
+ *   Set policy for stream in to prevent stop by another operation when current operation is more important.
```

Comment:
Set policy to prevent the current stream in from being stopped by another operation when the current operation is more important.

Reply:
done